hat/assets/js/apps/Iaso/domains/userRoles/components/CreateEditUserRole.tsx

```diff
@@ -98,43 +101,60 @@ export const CreateEditUserRole: FunctionComponent<Props> = ({
         setUserRolePermissoins(newPermissions);
         setFieldValue('permissions', newPermissions);
     };
+
+    useEffect(() => {
+        if (!open) {
+            setInfoOpen(true);
+        }
+    }, [closeDialog, open]);
```

Comment:
just this one looks weird

Reply:
I think this could be avoided by only calling `closeDialog` in the `onSuccess` callback (line 55) and adding

```javascript
 if (!open) {
            setInfoOpen(true);
        }
```

in the callback itself. To be tested, but it would be cleaner